apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -164,4 +167,16 @@
             public static boolean TRACE_DSL = false;
         }
     }
+
+    public static class Customize {
+        /**
+         * Custom enhancement class configuration file path, recommended to use an absolute path.
+         */
+        public static String ENHANCE_FILE = "";
+
+        /**
+         * Some information after custom enhancements, this configuration is used by the custom enhancement plugin.
+         */
+        public static Map<String, Object> CONTEXT = new HashMap<String, Object>();
```

Comment:
The reason you put the CONTEXT here, is this for avoiding classloader isolation issue? If SO, add this to comments.

Reply:
As you said for avoiding classloader isolation.